course_discovery/apps/edly_discovery_app/api/v1/constants.py

```diff
@@ -17,3 +17,5 @@
     'partner_name',
     'partner_short_code',
 ]
+
+DEFAULT_COURSE_ID = 'course-v1:{}+DEMO1+DEMO2'
```

Comment:
since course id are different for dev, stage and prod. we need to add it in yml file and import it in production settings

Reply:
this is the destination course id wouldn’t it be the same?